plugins/generator-1.16.4/forge-1.16.4/procedures/compare_mcitems_oredictionary_exact.java.ftl

```diff
@@ -0,0 +1,2 @@
+<#include "mcitems.ftl">

```

Comment:
I suggest you add non _exact variant of this procedure to this PR too (block tags possibly too).

Reply:
But I will have to make it because there is no `compare_mcitems_oredictionary` procedure, is there?

Reply:
Forget, what I just said, there is one, but I must change something and right now, I'm more focused on just moving procedures that doesn't need change between 1.15.2 and 1.16.4. I'll do a future PR where I add all the stuff that requires change.

Reply:
Ok, tyt. Actually _exact are legacy variants and non _exact are used, but _exact variants are still needed for older workspaces to work.

Reply:
It would be useless to release an update/snapshot until all 1.15.2 procedure blocks are ported to 1.16.4

Reply:
Indeed, a snapshot is planned after 1.16.4 support is complete.

Reply:
I noticed this procedure block does not compile at all

Reply:
I fixed this now.